introduction.tex

```diff
@@ -1,2 +1,40 @@
+SymPy is a full featured computer algebra system (CAS) written in the Python
+programming language. It is open source, licensed under the extremely
+permissive 3-clause BSD license, which allows anyone to reuse SymPy or its source code,
+even for commercial purposes. SymPy was started by Ond\v{r}ej \v{C}ert\'{\i}k
+in 2005, and it has since grown into a large open source project, with over
+500 contributors.
```

Comment:
Maybe Ondrej's name shouldn't be mentioned, but I did want to highlight the community.